calculate_average_karthikeyan97.sh

```diff
@@ -16,4 +16,14 @@
 #
 
 JAVA_OPTS="-Xms20480m -Xmx40960m "
-java $JAVA_OPTS --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_karthikeyan97
+
+if [ -f target/CalculateAverage_karthikeyan97_image ]; then
+    #echo "Picking up existing native image 'target/CalculateAverage_karthikeyan97_image', delete the file to select JVM mode." 1>&2
+    target/CalculateAverage_karthikeyan97_image -Xms20480m -Xmx40960m
+else
+    JAVA_OPTS="--enable-preview"
+    #echo "Chosing to run the app in JVM mode as no native image was found, use prepare_karthikeyan97.sh to generate." 1>&2
+    java -Xms20480m -Xmx40960m --class-path target/average-1.0.0-SNAPSHOT.jar dev.morling.onebrc.CalculateAverage_karthikeyan97
```

Comment:
Running with JVM aborts with 

java.lang.UnsupportedClassVersionError: Preview features are not enabled for dev/morling/onebrc/CalculateAverage_karthikeyan97 (class file version 65.65535). Try running with '--enable-preview'

Reply:
Yeah, I thought I had pointed that out before, but somehow my comment got lost.

@karthikeyan97 please make sure that _./test.sh_ passes without differences and that the output for 1B rows file matches that one of the baseline (or those currently fastest entries to be a bit quicker).